docs/conf/prague/2018/news/thanks-recap.rst

```diff
@@ -0,0 +1,62 @@
+
+Prague conference highlights and thanks to the community!
+---------------------------------------------------------
+
+Hello everyone,
+
+As we are wrapping up the Prague conference, we wanted to send out a quick email to thank everyone for helping to make our fifth European conference such a success.
+Whether you joined us in person or watched from afar, we thought you might like to get some of the highlights!
+
+Conference recap
+================
+
+At over 250 people, this year's conference was our biggest European event to date!
+We had the usual mix of main stage and lightning talks, unconference sessions, writing sprints, and social events.
+We were also pleased with the success of some new experiments, including livestreaming the talks, running real-time captioning for the main stage, and hosting an in-person job fair.
+We received a lot of positive feedback on all of these and are excited to carry them forward to future events.
+
+We also had some interesting feedback on some things we might improve for next time.
+The writing day, for example, picked up a lot of interest this year, and we're now looking at providing more in-depth guidance on how to get the most out of the sessions.
+
+Overall, we were so excited to see another conference come together and see our awesome community gather out in the physical world.
+
+Videos, photos, and write-ups
+=============================
+
+Videos of all the talks are now available `in our video archive <http://www.writethedocs.org/videos/prague/2018/>`__.
+Thanks to `Here to Record <https://heretorecord.com/>`__ for doing such a great job with these.
+If you gave a lightning talk, we're working to get them on our site, but in the meantime you can see them on `YouTube <https://www.youtube.com/watch?v=oXmrFoEEf3A&list=PLZAeFn6dfHplRZcYDQjST22bAVeeWML4d>`__ for now.
+(You can also subscribe to our `YouTube channel <https://www.youtube.com/channel/UCr019846MitZUEhc6apDdcQ>`_
+so you don't miss any future content!)
+
+We also have photos from the event available on `Flickr <https://www.flickr.com/photos/writethedocs/albums/72157695294209550>`_, thanks to our wonderful photographer `Kay Smoljak <https://twitter.com/goatlady>`_!
+We really love the photos each year, and you're welcome to use them for non-commercial purposes, as per our `Creative Commons License <https://creativecommons.org/licenses/by-nc-sa/2.0/>`_.
+
+And finally, several community members have been writing up their experiences at the conference and sharing them in the `wtd-conferences channel on the Write the Docs Slack <https://writethedocs.slack.com/archives/wtd-conferences>`__. If you missed the conference (or are just curious about how your experience compares to others'), check out the discussions there.
+
+Code of Conduct transparency report
+===================================
+
+As with any Write the Docs event, this conference was covered by our `community Code of Conduct <http://www.writethedocs.org/code-of-conduct/>`__. 
```

Comment:
We may want to add a bit more on what we did, especially because there were no reports. I suggest inserting something like:

```
We took a number of specific steps:

* Every participant agreed to the Code of Conduct in advance.
* The CoC, or a summary, was included on the conference website, conference booklet, and in posters around the conference venue.
* The CoC was repeatedly mentioned in announcements on the stage.
* We specifically stressed that the CoC applies to all conference spaces including the party, and therefore attendees should drink responsibly.
* We encouraged all attendees to report any incident, even if they were not sure whether this was a violation.
```